<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admiralSelectDialog</class>
 <widget class="QDialog" name="admiralSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>19</y>
     <width>371</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QRadioButton" name="l4Radio">
          <property name="text">
           <string>Level 4</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">radioButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="l3CostLabel">
          <property name="text">
           <string>cost3</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="l2Radio">
          <property name="text">
           <string>Level 2</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">radioButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="l4CostLabel">
          <property name="text">
           <string>cost4</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="l5CostLabel">
          <property name="text">
           <string>cost5</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="l3Radio">
          <property name="text">
           <string>Level 3</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">radioButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="l2CostLabel">
          <property name="text">
           <string>cost2</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QRadioButton" name="l5Radio">
          <property name="text">
           <string>Level 5</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">radioButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="l0Radio">
          <property name="text">
           <string>No Admiral</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">radioButtonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="doneButton">
      <property name="text">
       <string>Set Admiral</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="radioButtonGroup"/>
 </buttongroups>
</ui>
